packages/css/test/index.js

```diff
@@ -230,6 +230,21 @@ test('handles responsive variants', () => {
   })
 })
 
+test('returns multiple variants from theme', () => {
```

Comment:
I'd suggest adding a few more tests for when the value of `variants` *isn't* an array and maybe one to demonstrate how using both `variant` and `variants` together behaves